.github/workflows/release.yml

```
@@ -91,5 +97,5 @@ jobs:
             VERSION=${{ github.event.inputs.version_tag }}
             COMMIT_SHA=${{ github.sha }}
           tags: |
-            ghcr.io/erpc/erpc:latest
-            ghcr.io/erpc/erpc:${{ github.event.inputs.version_tag }}
+            ghcr.io/${{ steps.tag_param.outputs.repo }}:latest
+            ghcr.io/${{ steps.tag_param.outputs.repo }}:${{ github.event.inputs.version_tag }}
```

How about directly using this here?

```suggestion
            ghcr.io/${{ github.repository }}:latest
            ghcr.io/${{ github.repository }}:${{ github.event.inputs.version_tag }}
```

---

I'll merge for now since you've tested this.

---

Hi @aramalipoor as mentioned in the PR description this wouldn't work because Docker tags must be all lower case and `github.repository` preserves the original casing.

---

For example in my fork it would fail because it wants to use `ghcr.io/xJonathanLEI/erpc:latest` which is not a legal tag.

---

Ah I just got the point for string manipulation, for lower-casing 🤝